lib/WasmReader/WasmByteCodeGenerator.cpp

```diff
@@ -690,8 +698,10 @@ WasmBytecodeGenerator::EmitCall()
         switch (info.type)
         {
         case WasmTypes::F32:
-            Assert(wasmOp != wnCALL_IMPORT);
-            // REVIEW: support FFI call with f32 params?
+            if (wasmOp == wnCALL_IMPORT) 
+            {
+                throw WasmCompilationException(_u("External calls with float argument NYI"));
```

Comment:
I guess there's no real reason to forbid, but we would still convert to double (when creating the js number) before calling.